bin/lib/aws/translation-api-limit.js

```diff
@@ -105,25 +103,24 @@ function updateApiLimitUsed(articleCharacters) {
 	});
 }
 
-function constructUpdateExpression(articleCharacters) {
+function constructUpdateExpression(translators) {
 	let updateExpression = 'SET updatedAt = :updatedAt ADD ';
 
-	Object.keys(articleCharacters).forEach((provider, index) => {
-		updateExpression =
-			updateExpression + `${index === 0 ? '' : ','} ${provider} :${provider}`;
-	});
+	for (let i = 0; i < translators.length; ++i) {
```

Comment:
Why the switch from ForEach to for loop in translation-api-limit.js? Was it for the performance gain?

Reply:
Ah, nope, written by 2 different devs would be the reason 😊

Reply:
ah ok :)